### ISSUE TITLE
Illegal reflective access with groovy 3.0.0-beta2 on jdk 12.

### ISSUE THREAD

**Issue by hazendaz** [spotbugs/spotbugs-maven-plugin#132]
Issue can be found by running our integration tests and looking at the 'build.log' for check-fail test.

See further details on history of fixing these via #103 along with this report.  While we cannot actually fix this, this is simply a placeholder until groovy fixes the issue to help assist users.

Below is the output seen.

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.reflection.ReflectionUtils (file:/C:/spotbugs-maven-plugin/target/local-repo/org/codehaus/groovy/groovy/3.0.0-beta-2/groovy-3.0.0-beta-2-indy.jar) to method java.util.Collections$UnmodifiableCollection.toString()
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.ReflectionUtils
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by markkolich** [comment]
+1 here on JDK 13

**Comment by cgoeller** [comment]
The same issue occurs on jdk11

**Comment by hazendaz** [comment]
While this issue is groovy related, changes were performed on our logging within #173 and therefore this appears cleared now from last places that I saw.  I am working on releasing support tied to spotbugs 4.0.0.  I'm closing this now.  If this shows up again, feel free to call out where.
